README.md

```diff
@@ -35,9 +35,6 @@ In order to publish to Maven Central (aka the Central Repository or just Central
 
 ```groovy
 nexusPublishing {
-    repositories {
-        sonatype()
-    }
```

Comment:
@szpak I wonder if this method should be deprecated or the URLs changed to the new ones.